newrelic_beego.go

```diff
@@ -87,6 +87,26 @@ func StartTransaction(ctx *context.Context) {
 	ctx.Input.SetData(newRelicTransaction, tx)
 }
 
+// shouldSkip decides if given path matches against declared skip paths
+// Support both exact matches and wildcard matches
+func shouldSkip(skipPaths map[string]bool, path string) bool {
+	_, exactMatch := skipPaths[path]
+
+	if exactMatch {
+		return true
+	}
+
+	for skipPath := range skipPaths {
+		if strings.HasSuffix(skipPath, "*") {
```

Comment:
Thant's not good, since most APIs start with /api/<env>, and I want to skip it too.
Just use contains here, without looking for *, it will simplify everything and work just fine